### PR TITLE
Removed deprecated webapp configs and the code that used them.

### PR DESCRIPTION
**Pull request by Nishith**

### CODE REVIEW

**Comment by Nishith** [comment]
https://issues.cask.co/browse/CDAP-4961

**Comment by anwar6953** [comment]
Please update:
cdap-docs/tools/cdap-default/cdap-default-deprecated.xml
cdap-docs/tools/cdap-default/cdap-default-exclusions.txt
cdap-api/src/main/java/co/cask/cdap/api/webapp/WebappSpecification.java

**Comment by Nishith** [comment]
@awholegunch is changing the docs.

**Comment by anwar6953** [comment]
co.cask.cdap.internal.app.runtime.distributed.DistributedWebappProgramRunner

**Comment by anwar6953** [comment]
ProgramType.Webapp

**Comment by anwar6953** [comment]
package co.cask.cdap.internal.app.runtime.webapp;

**Comment by anwar6953** [comment]
Constants.Router.WEBAPP_DISCOVERY_NAME

**Comment by anwar6953** [comment]
Networks.normalizeWebappDiscoveryName

**Comment by anwar6953** [comment]
LGTM

**Comment by awholegunch** [comment]
Running a quick build to check: http://builds.cask.co/browse/CDAP-DQB200-1

It will probably fail due to other changes outside of this PR.

**Comment by awholegunch** [comment]
Fails, but unrelated to this PR. http://builds.cask.co/browse/CDAP-DQB200-1/log

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -1731,14 +1731,7 @@
...
     <description>
-    </description>
-  </property>
-
-  <property>
-    <name>router.ssl.webapp.bind.port</name>
-    <value>20443</value>
-    <description>
-      CDAP Router service bind port for webapp for HTTPS (deprecated)
+      CDAP Router service port to which CDAP UI connects for HTTPS
```
Comment (Nishith) [inline cdap-common/src/main/resources/cdap-default.xml]: This is not just for UI, REST calls will go here as well.

**Comment by sreevatsanraman** [comment]
LGTM

**Pull request merged**